no-computer.md

```diff
@@ -11,41 +11,106 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A group of people on campus regularly play a large game of Diplomacy.
+The board games involve the players sending in instructions for their units
+on the board.
+These instructions are collected then played out simultaneously.
+This causes a lot of conflicts between their instructions which are then 
+resolved based on the game's rules.
+Further complications arise when players send invalid instructions.
+
+A program that can take in these instructions and then validate moves and
+resolve conflicts would speed up this process.
+As of now, it takes 5 to 10 minutes for people to manage this process.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This program would need to take in inputs from the players.
+A DSL would be an appropriate method to take in these instructions.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I like boardgames and I'm somewhat interested in boardgame design. 
+This project builds on a homebrewed version of Diplomacy, so I'm excited to
+contribute to a project that already exist and needs help.
```

Comment:
So this project would be to build upon an existing DSL right? Would that entail creating an internal DSL within the DSL?